presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -99,14 +104,13 @@ Presto restart. The refresh period is specified in the ``etc/access-control.prop
 
    security.refresh-period=1s
 
+.. _catalog_rules:
 
 Catalog Rules
 -------------
 
-These rules govern the catalogs particular users can access. The user is
```

Comment:
It's fine to repeat the "top to bottom" information here. A user might visit this section directly, without having read the introduction at the top. It's one extra sentence, so repeating it doesn't hurt readability, and serves to reinforce the concept.